services/QuillLMS/lib/tasks/feedback_history.thor

```diff
@@ -0,0 +1,163 @@
+# frozen_string_literal: true
+
+require 'thor'
+require './config/environment'
+
+class FeedbackHistory < Thor
```

Comment:
Have you hit any issues with this being the same name as the model? Classes in this folder aren't autoloaded in the app, so I think it would be ok.

Reply:
I didn't hit any issues when running on staging, but I will change it anyways, just to be safe.